src/frontend/src/app/scheduled-process/scheduled-process.component.ts

```diff
@@ -7,48 +7,119 @@ import { SchedulesService } from '../services/index';
+    this.getSchedules()
+    this.getSchedulesProcesses()
+    this.getLatestTasks()
+  }
+
+  public getScheduleType(): void {
+    this.schedulesService.getScheduleType().
+      subscribe(
+      data => {
+        this.scheduleType = data.schedule_type
+        console.log(this.scheduleType)
+      },
+      error => { console.log("error", error) })
   }
 
-  public getSchedules():void {
-    this.schedulesData = [];
+  public getSchedules(): void {
+    this.scheduleData = [];
     this.schedulesService.getSchedule().
       subscribe(
       data => {
-        this.schedulesData = data.schedules;
-        console.log("This is the getSchedules ",  data.schedules);
+        this.scheduleData = data.schedules
+        console.log("This is the getSchedule ", data.schedules)
       },
       error => { console.log("error", error) });
   }
 
-  public getSchedulesProcesses():void {
-    this.schedulesProcessData = [];
+  public getSchedulesProcesses(): void {
+    this.scheduleProcess = [];
     this.schedulesService.getScheduledProcess().
       subscribe(
       data => {
-        this.schedulesProcessData = data.processes;
-        console.log("This is the getSchedulesProcesses ",  this.schedulesProcessData);
+        this.scheduleProcess = data.processes
+        console.log("This is the getScheduleProcess ", this.scheduleProcess)
       },
       error => { console.log("error", error) });
   }
 
-  public getLatestTasks():void {
+  public getLatestTasks(): void {
     this.tasksData = [];
     this.schedulesService.getLatestTask().
       subscribe(
       data => {
         this.tasksData = data.tasks;
-        console.log("This is the tasks ",  data.tasks);
+        console.log("Tasks data", data.tasks)
       },
-      error => { console.log("error", error) });
+      error => { console.log("error", error) })
+  }
+
+  public setType(value){
+      this.selectedType = value;
+  }
+
+  public createSchedule() {
+    let schedule_name = <HTMLInputElement>document.getElementById("name")
+    let schedule_process = <HTMLSelectElement>document.getElementById("process")
+    let schedule_type = <HTMLSelectElement>document.getElementById("type")
+    let repeat_interval = <HTMLInputElement>document.getElementById("repeat")
+    let exclusive_state = <HTMLInputElement>document.getElementById("exclusive")
+    
+    if(this.selectedType == 2) {
+        var scheduler_day_field = <HTMLSelectElement>document.getElementById("day")
+        var scheduler_time_field = <HTMLInputElement>document.getElementById("time")
+        var day = scheduler_day_field.value
+        var time = scheduler_time_field.value
+        var scheduled_time = time != '' ? this.converTime(time):0
+    }
+    var repete_time = repeat_interval.value != '' ? this.converTime(repeat_interval.value):0
+    this.invalidRepeat = this.between(repete_time) 
+    this.invalidTime =  this.between(scheduled_time)
+    if (this.invalidRepeat || this.invalidTime) {
```

Comment:
I really doubt that we need these invalidRepeat, invalidTime var for `this.`
and method should be either not_bwtween or return NOT of current computation...

Reply:
invalidRepeat and invalidTime values are used for display time field range validation message  on html page only. That is the reason we need to make them class level var.